java/README.md

```diff
@@ -0,0 +1,74 @@
+# Java KvikIO Bindings
+
+## Summary
+These Java KvikIO bindings for GDS currently support only synchronous read and write IO operations using the underlying CuFile API. Support for batch IO and asynchronous operations are not yet supported.
```

Comment:
For consistency, use a lowercase c in cuFile. That matches the official docs. Please apply this everywhere in your PR.
```suggestion
These Java KvikIO bindings for GDS currently support only synchronous read and write IO operations using the underlying cuFile API. Support for batch IO and asynchronous operations are not yet supported.
```